docs/cloud/dev/spryker-cloud-commerce-os/adding-databases-for-stores.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Adding databases for stores
+description: Learn how to deploy a dedicated database for a store
+template: howto-guide-template
+---
+
+This document describes how to deploy a dedicated database for a store.
+
+
+## Define a database
+
+1. In the needed deploy file, define the new database:
+```yaml
+...
+
+regions:
+    EU:
+        services:
+            mail:
+                sender:
+                    name: Spryker No-Reply
+                    email: no-reply@spryker.local
+            databases:
+                eu-region-existing-database:
```

Comment:
Why the database name contains `existing`?
Maybe it's better to name them:
database-de and database-at?